notebooks/community/feature_store/mobile_gaming_feature_store.ipynb

```diff
@@ -0,0 +1,2274 @@
+{
```

Comment:
<p>Is this section using Feature Store online serving or Endpoint online serving? </p><p> </p><p>Is the "Simulate online prediction requests" below suppose to be the <code>read</code> result from entity_type?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>

Reply:
<p>The section uses Feature Store online serving in conjunction with Vertex AI Endpoint. For instance, it simulates a user with <code>DE346CDD4A6F13969F749EA8047F282A</code> ID who is based in US, uses IOS with english as default language of the application. With that said, </p><p> </p><ol><li>the ID is used in <code>read</code>  features of customer_entity_type from Vertex AI Feature Store</li><li>Those features are joined with <code>country</code>,  <code>operating_system</code>  and <code>language</code> , passed to the endpoint by generating predictions</li></ol>